docs/reference/migration/apis/shared-migration-apis-tip.asciidoc

```diff
@@ -0,0 +1,4 @@
+TIP: These APIs are designed for indirect use by {kib}'s **Upgrade Assistant**.
+We strongly recommend you use the **Upgrade Assistant** to upgrade from
+{prev-major-last} to {version}. For upgrade instructions, refer to
```

Comment:
> from {prev-major-last} to {version}

In the 7.17 backport, I'll update this to "from 7.17 to 8.0 and later versions."